docs/finality-provider-operation.md

```diff
@@ -939,4 +943,43 @@ For Finality Provider:
 
 > 🔒 **Security Note**: While database files can be recreated, loss of private keys in the keyring directories is **irrecoverable** and will result in permanent loss of your finality provider position and accumulated rewards.
 
-> 💡 **Future Updates**: The ability to recreate the `fpd.db` from chain state will be offered in future releases, but keyring backup remains critical.
+### 7.3 Recover finality-provider.db
+
+The `finality-provider.db` file contains both the finality provider's running
+status and the public randomness merkle proof. Either information loss
+compromised will lead to service halt, but they are recoverable.
+
+#### 7.3.1 Recover local status of a finality provider
```

Comment:
If there is a loss in the db, the steps are

- Stop the fpd
- recover `fpd recover-rand-proof`
- Start fpd 
- Create finality provider `fpd create-finality-provider`
- Stop the fpd
- Start the fpd 

Is this expected?

Reply:
I think if the db file is lost, fpd cannot be started due to missing the fp object. So I think the first step should be start `fpd` and run `fpd create-finality-provider`. Then `stop fpd` and run `fpd recover-rand-proof`. Finaly, restart the `fpd`